src/main/scala/org/hammerlab/guacamole/reads/Read.scala

```diff
@@ -61,14 +61,14 @@ trait Read {
   val isDuplicate: Boolean
 
   /** Is this read mapped? */
-  final val isMapped: Boolean = getMappedReadOpt.isDefined
+  final val isMapped: Boolean = this match {
```

Comment:
I was thinking that it would improve performance since we would serialize less, but I guess we're manually controlling what gets kryo-serialized so nvm